code/WorkInProgress/nadir_antenna.dm

```diff
@@ -6,16 +6,40 @@ var/global/obj/machinery/communications_dish/transception/transception_array
 //Cost to follow through on the transception, charged against grid in grid units of power
 #define ARRAY_TELECOST 2500
 
```

Comment:
Other general comment: can you put a description of the different light meanings somewhere, either in a code comment or on a piece of paper? Ideally someone can make a graphic or a gif or whatever eventually but having the description around would be nice.

Reply:
I plan to update #11689 with a description of the indicators, but codeside commentary also makes sense. Will do both.

Reply:
Codeside done.